.github/workflows/codeql.yaml

```diff
@@ -0,0 +1,35 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+  schedule:
+    - cron: "0 7 * * 1" # Mondays at 7:00 AM
```

Comment:
Is this UTC or PST?

Reply:
It's UTC

Reply:
Do we want to move to PST morning?

Reply:
I've no strong opinion on that. If you would like me to change it, I can do that.

Reply:
Let's do it!

Reply:
done!